src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -24,39 +25,32 @@ public class FormulaValueJSON
         /// <summary>
         /// Convenience method to create a value from a json representation. 
         /// </summary>
-        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null, bool numberIsFloat = false)
+        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null, bool numberIsFloat = false, ResourceManager resourceManager = null)
```

Comment:
>ResourceManager resourceManager = null [](http://example.com/codeflow?start=123&length=38)

API breaking change

Reply:
Inevitable if we want to manage blobs as we can't create those without a resource manager